### PR TITLE
CHANGE: remove bintray from docs.

### PR DESCRIPTION
**Pull request by andreaspeters**
@qianzhangxa Like we talked, here my PR to remove bintray from the docs.

### CODE REVIEW

File: docs/binary-packages.md [hunk 1]
```diff
@@ -5,28 +5,9 @@ layout: documentation
...
-
-    $ sudo mv /tmp/bintray-mesos-el.repo /etc/yum.repos.d/bintray-mesos-el.repo
-
-    $ sudo yum update
-
-    $ sudo yum install mesos
-
-The above instructions show how to install the latest version of Mesos for RHEL 7.
-Substitute `baseurl` the with the appropriate URL for your operating system.
+Download and install the latest stable binary: [Downloads](https://mesos.apache.org/downloads/)
```
Comment (qianzhangxa) [inline docs/binary-packages.md:10]: Can we change this to the below?

> Download and install [the latest stable Mesos binaries](https://mesos.apache.org/downloads/).

File: docs/binary-packages.md [hunk 2]
```diff
@@ -5,28 +5,9 @@ layout: documentation
...
 
 ## Start Mesos Master and Agent.
```
Comment (qianzhangxa) [inline docs/binary-packages.md:12]: Not yours, but I think we do no need the trailing period.

Comment (andreaspeters) [reply docs/binary-packages.md:12]: Sorry, I don't understand. :sweat_smile:  Do you mean we don't need "Start Mesos Master...."?

Comment (qianzhangxa) [reply docs/binary-packages.md:12]: I mean it should be `## Start Mesos Master and Agent` rather than `## Start Mesos Master and Agent.`.

Comment (andreaspeters) [reply docs/binary-packages.md:12]: Ah ok I got it. :-) Thanks.

**Comment by qianzhangxa** [comment]
@andreaspeters Thanks for your contribution!

**Pull request merged**